docs/accessControl.md

```diff
@@ -198,6 +199,32 @@ To block students from viewing closed assessment scores, set `"showClosedAssessm
 
 The `showClosedAssessment` access rule restriction is only useful in conjunction with [disabling real-time grading](assessment.md#disabling-real-time-grading) and setting `"showClosedAssessment": false`.
 
+## Submittable assessments
+
+The `submittable` access rule restriction is useful for allowing students to see what assessments they have coming up. If `submittable` is set to `false`, students can see the assessment on the Assessments page, but they cannot create a new assessment instance or submit answers to questions. If an assessment is currently not submittable but will be in the future, students can see when the assessment will become submittable by looking at the `Available credit` column on the Assessments page. Assessments are submittable by default.
+
+**Note**: if `submittable` is set to `false` in an access rule, the available `credit` for that access rule must be 0.
+
+An example of the `submittable` access rule restriction is shown below:
+
+```json
+"allowAccess": [
+    {
+        "startDate": "2015-02-16T00:00:01",
+        "endDate": "2015-02-16T23:59:59",
+        "timeLimitMin": 50,
+        "credit": 100
+    },
+    {
+        "startDate": "2015-01-01T00:00:01",
+        "endDate": "2015-02-15T23:59:59",
+        "submittable": false
+    }
+]
+```
+
+In the example above, from January 1st to February 15th, students can see the assessment on the Assessments page but cannot begin the assessment. If students try to click into the assessment during this time, they will see a message saying that the assessment will be available on February 16th. The first access rule applies on February 16th, and since `submittable` is `true` (by default), students can start the assessment and submit answers to questions on that day.
```

Comment:
```suggestion
In the example above, from January 1st to February 15th, students can see the assessment on the Assessments page but cannot begin the assessment. They will see a message saying that the assessment will be available on February 16th. The first access rule applies on February 16th, and since `submittable` is `true` (by default), students can start the assessment and submit answers to questions on that day.
```